ydb/public/lib/ydb_cli/commands/sqs_workload/http_client.cpp

```diff
@@ -0,0 +1,102 @@
+#include "http_client.h"
+#include "consts.h"
+
+#include <aws/core/client/ClientConfiguration.h>
+#include <aws/core/utils/DateTime.h>
+
+#include <fmt/format.h>
+
+namespace NYdb::NConsoleClient {
+
+    namespace {
+
+        const TString kLatencyMetric = "RequestLatency";
```

Comment:
```suggestion
        const TStringBuf kLatencyMetric = "RequestLatency";
```

и по стилю костнанты через капитализацию, а не `k`